presto-main/src/test/java/com/facebook/presto/operator/TestWindowOperator.java

```diff
@@ -80,11 +80,11 @@
     );
 
     private static final List<WindowFunctionDefinition> LAG = ImmutableList.of(
-            window(new ReflectionWindowFunctionSupplier<>("lag", VARCHAR, ImmutableList.of(VARCHAR, BIGINT, VARCHAR), LagFunction.class), VARCHAR, UNBOUNDED_FRAME, 1, 3, 4)
+            window(new ReflectionWindowFunctionSupplier<>("lag", VARCHAR, true, ImmutableList.of(VARCHAR, BIGINT, VARCHAR), LagFunction.class), VARCHAR, UNBOUNDED_FRAME, 1, 3, 4)
```

Comment:
By default value WFs are `RESPECT NULLS` so please don't change that test to non-default behavior. Add new tests for non-default behavior to cover your newly added feature, instead.